core/src/input/input.stories.tsx

```diff
@@ -0,0 +1,58 @@
+import { _Story } from "../_story";
+import { Input } from "./input";
+
+export default {
+	title: "Components/Input",
+	component: Input,
+	argTypes: {
+		type: _Story.arg(null),
+		style: _Story.arg(Input.styles),
+		size: _Story.arg(Input.sizes),
+		maxLength: _Story.arg("number"),
+		disabled: _Story.arg("boolean"),
+		readOnly: _Story.arg("boolean"),
+		placeholder: _Story.arg(null),
+		icon: _Story.arg(null),
+
+		value: _Story.arg(null),
+		setValue: _Story.arg(null),
+
+		list: _Story.arg(null),
+
+		id: _Story.arg(null),
+		autoFocus: _Story.arg(null),
+		autoSelect: _Story.arg(null),
+		"aria-label": _Story.arg(null),
+		"aria-labelledby": _Story.arg(null),
+
+		onBlur: _Story.arg(null),
+		onFocus: _Story.arg(null),
+		onKeyPress: _Story.arg(null),
+		onKeyUp: _Story.arg(null),
+		onKeyDown: _Story.arg(null),
+		onClick: _Story.arg(null),
+		onChange: _Story.arg(null),
```

Comment:
Do we need to list these "null" params? I thought they will show anyway

Reply:
Without "null" params, it will show the text area, like this:
![image](https://user-images.githubusercontent.com/39718754/111447658-3a421c80-8740-11eb-8294-2925d6bf0524.png)

Reply:
oh that's bad then. Let's keep "null"